includes/html/graphs/graph.inc.php

```diff
@@ -24,6 +24,7 @@
 $output   = (!empty($vars['output']) ? $vars['output'] : 'default');
 $from = (isset($vars['from']) ? $vars['from'] : time() - 60 * 60 * 24);
 $to   = (isset($vars['to']) ? $vars['to'] : time());
+$graph_type = (isset($vars['graph_type']) ? $vars['graph_type'] : $config['webui']['graph_type']);
```

Comment:
Could you convert `$config['webui']['graph_type']` to `\LibreNMS\Config::get('webui.graph_type');`?